src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java

```diff
@@ -23,10 +23,10 @@ public TrucodeAuthenticator(TrusonaApi trusona) {
     Principal principal = null;
 
     TrusonaficationResult result = trusona.createTrusonafication(Trusonafication.essential()
-      .truCode(trucodeId)
-      .action(action)
-      .resource(resource)
-      .build());
+                                                                   .truCode(trucodeId)
```

Comment:
Did you use a linter for this? I don't think chained methods should be justified with the first opening paren. Thats usually done for method args.

Reply:
I did use whatever the default is in in intellij

Reply:
my intellij config is set to [this](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)